templates/pages/reset-password.jsx

```diff
@@ -28,32 +29,47 @@ var ResetPassword = React.createClass({
   render: function() {
     var emailText = "We've emailed you instructions for creating a new password.";
     var linkQuery = {};
+    var content;
+
     linkQuery.client_id = this.state.queryObj.client_id;
     linkQuery.state = this.state.queryObj.state;
     linkQuery.scopes = this.state.queryObj.scopes;
     linkQuery.response_type = this.state.queryObj.response_type;
-    return (
-      <div>
-        <Header origin="Reset Password" className="desktopHeader" redirectQuery={linkQuery} />
-        <Header origin="Reset Password" className="mobileHeader" redirectLabel="Signup" redirectPage="signup" redirectQuery={linkQuery} mobile />
 
+    if (this.state.resetSuccess) {
+      content = (
+        <div className="centerDiv smallWrapper">
+          <PasswordResetSuccess android="true"/>
```

Comment:
the flag changes the message displayed. We can assume android to be true here because otherwise we'd have been redirected to where it's rendered without the `android=true` bits (that's here: https://github.com/mozilla/id.webmaker.org/pull/369/files#diff-bd3f8113697433376585269d847be735R71 )